packages/website/src/components/RecruitmentModal/RecruitmentModal.tsx

```diff
@@ -24,6 +26,8 @@ interface RecruitmentModalProps {
 export const RecruitmentModal = ({ isOpen, config, onClose, modalType }: RecruitmentModalProps) => {
   const modalContentPaddings = useBreakpointValue({ base: '32px 16px 16px 32px', md: '56px 64px 64px 64px' } as const);
   const modalBodyMarginBottom = useBreakpointValue({ base: '16px', lg: '32px' } as const);
+  const modalCloseIconSize = useBreakpointValue({ base: 'sm', md: 'lg' } as const);
+  const modalHeaderTextMargin = useBreakpointValue({ base: '32px', md: '60px' } as const);
```

Comment:
🤔 For large screens we use only values that are divisible by 8, so 64px for the `md` margin will be better